task.dart

```diff
@@ -0,0 +1,118 @@
+import 'dart:io';
+
+void main() {
+  bool flag = true;
+  var myList = [];
+  print("How many tasks do you want to do : \n");
+  int numberOfTask = int.parse(stdin.readLineSync()!);
+
+  print("Enter your tasks to do: \n ");
+  for (int i = 0; i < numberOfTask; i++) {
+    var taskToDo = stdin.readLineSync();
+    myList.add(taskToDo);
+  }
+
+  do {
+    print("What do you want \n");
+    print("1. Add task");
+    print("2. Edit task");
+    print("3. Delete task");
+    print("4. complete task");
+    print("5. Exit");
+    print("Write down the number of your choice: \n ");
+
+    int choice = int.parse(stdin.readLineSync()!);
+
+    switch (choice) {
+      case 1:
+        {
+          // to add task
+          print("Enter the task to add: ");
+          var newTask = stdin.readLineSync();

```

Comment:
Usage of var should be limited as Dart is a type safe language. Specifying a type can help avoid some errors.
Also, Double check if empty
```suggestion
          var newTask = stdin.readLineSync();
          if(newTask.isEmpty)
          {
            print("Cannot add empty task");
          }
```